projects/storefrontlib/cms-components/product/product-intro/product-intro.component.ts

```diff
@@ -28,6 +29,8 @@ export class ProductIntroComponent {
   product$: Observable<Product | null> =
     this.currentProductService.getProduct();
 
+  protected featureConfigService = inject(FeatureConfigService);
```

Comment:
Lets change this to `private` as we don't want to expose it as a property to users of our libs. This makes it easier to remove later when we want to introduce breaking changes ;)

Reply:
I've changed it.